brainscore/benchmarks/imagenet_c.py

```diff
@@ -62,28 +63,52 @@ class Imagenet_C_Category(BenchmarkBase):
     impulse noise [1-5]
     """
     noise_category_map = {
-        'noise': ['gaussian_noise', 'shot_noise', 'impulse_noise'],
-        'blur': ['glass_blur', 'motion_blur', 'zoom_blur', 'defocus_blur'],
-        'weather': ['snow', 'frost', 'fog', 'brightness'],
-        'digital': ['pixelate', 'contrast', 'elastic_transform', 'jpeg_compression']
+        'noise'   : ['gaussian_noise', 'shot_noise', 'impulse_noise'],
+        'blur'    : ['glass_blur', 'motion_blur', 'zoom_blur', 'defocus_blur'],
+        'weather' : ['snow', 'frost', 'fog', 'brightness'],
+        'digital' : ['pixelate', 'contrast', 'elastic_transform', 'jpeg_compression']
     }
 
     def __init__(self, noise_category, sampling_factor=10):
         self.noise_category = noise_category
         self.stimulus_set_name = f'dietterich.Hendrycks2019.{noise_category}'
-
+        
         # take every nth image, n=sampling_factor.
-        stimulus_set = brainscore.get_stimulus_set(self.stimulus_set_name)[::sampling_factor]
-        self.stimulus_set = stimulus_set
+        self.stimulus_set = self.load_stimulus_set()[::sampling_factor]
+        print(f'Loaded local {self.noise_category} stimulus set')
         self.noise_types = self.noise_category_map[noise_category]
 
         ceiling = Score([1, np.nan], coords={'aggregation': ['center', 'error']}, dims=['aggregation'])
-        super(Imagenet_C_Category, self).__init__(identifier=f'dietterich.Hendrycks2019-{noise_category}-top1',
+        super(Imagenet_C_Category, self).__init__(identifier=f'dietterich.Hendrycks2019-{noise_category}-top1', 
                                                   version=2,
                                                   ceiling_func=lambda: ceiling,
                                                   parent='dietterich.Hendrycks2019-top1',
                                                   bibtex=BIBTEX)
 
+
+    def load_stimulus_set(self):
+        """
+        ImageNet-C is quite large, and thus cumbersome to download each time the benchmark is run.
+        Here we try loading a local copy first, before proceeding to download the AWS copy.
+        """
+        try:
+            print(f'Try loading local Imagenet-C {self.noise_category}')
+            category_path = os.path.join(
+                LOCAL_STIMULUS_DIRECTORY, 
+                f'image_dietterich_Hendrycks2019_{self.noise_category}'
+            )
+            loader = StimulusSetLoader(
+                csv_path=os.path.join(category_path, f'image_dietterich_Hendrycks2019_{self.noise_category}.csv'), 
+                stimuli_directory=category_path, 
+                cls=None
+            )
+
+            return loader.load()
+        
+        except OSError as error:
+            print(f'Excepted {error}. Attempting to access {self.stimulus_set_name} through Brainscore.')
```

Comment:
Could you use a logger instead? E.g.
```
import logging
_logger = logging.getLogger(__name__)
```
And then `_logger.debug("Expected ..")

This will allow us to write things out to disk instead of print statements that only go to the console